app/controllers/schools/choose_programme_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Schools::ChooseProgrammeController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
+
+  def show
+    school = @current_user.induction_coordinator_profile.schools.first
+    redirect_to helpers.profile_dashboard_url(@current_user) if school.chosen_programme?(Cohort.current)
+
+    @induction_choice_form = InductionChoiceForm.new
+  end
+
+  def create
+    @induction_choice_form = InductionChoiceForm.new(params.require(:induction_choice_form).permit(:programme_choice))
+
+    render :show and return unless @induction_choice_form.valid?
+
+    @cohort = Cohort.current
+    @school = current_user.induction_coordinator_profile.schools.first
+
+    @school_cohort = SchoolCohort.find_or_create_by!(
```

Comment:
That variable is unused, I think. All the more reason to use local variables when possible - RubyMine will tell you the variable is unused if it's local, but not if it's `@`.